### ISSUE TITLE
feature request https://charts.bitbo.io/long-term-power-law/

### ISSUE THREAD

**Issue by at1776** [BitboBTC/Feedback#27]
Hi it would be nice to have the halving events marked on this chart.  Please advise.

**Comment by JTuwiner** [comment]
We have this chart https://charts.bitbo.io/halving-progress/

Unfortunately, might be some time before we can add a chart that is the two of those combined.
